### ISSUE TITLE
Fixes for cycling soil analysis experiments

### ISSUE THREAD

**Issue by ClaraDraper-NOAA** [NOAA-EMC/GSI#593]
A number is fixes are required to run cycling experiments with the new EnKF soil analysis:

1. The soil moisture and temperature increments are added outside the GSI code (in UFS_UTILS). This means that the soil DA increments must always be written out, regardless of whether the increment or analysis is requested for the atmosphere. 

2. The enkf reads in the background states in gridio_gfs.f90. The variable names for the soil states need to be updated from slc[1234] to soill[1234] and stc[1234] to soilt[1234]. 

3. soil temperature (STC) is defined over the ocean (and holds some sort of ocean temp). soil moisture (SLC) is defined, and given a value of 1 over the ocean. The discontinuities in ensemble spread at the coast between STC/SLC-ocean and STC/SLC-land induce extreme values in the output of the ensemble smoothing routine (which can then crash the model, due to the extreme soil temperature gradients created). We may wish to develop a smoothing solution for land variables in the future, but for now the immediate solution is to not apply the smoothing to soil variables.

**Comment by ClaraDraper-NOAA** [comment]
@CoryMartin-NOAA 2 of 2 issues I need to merge before you can start soil analysis experiments.

**Comment by CoryMartin-NOAA** [comment]
@ClaraDraper-NOAA are these already done and just need merged in through PRs or is there development work that still needs to be done?

**Comment by ClaraDraper-NOAA** [comment]
I've already done them, and just need to merge them in.

**Comment by ClaraDraper-NOAA** [comment]
... will get to in this week, hopefully.

**Comment by CoryMartin-NOAA** [comment]
Got it, let me know how I can help

**Comment by RussTreadon-NOAA** [comment]
@ClaraDraper-NOAA , the GSI Handling Review team is working through GSI issues to see which, if any, may be closed.  May this issue be closed?

**Comment by ClaraDraper-NOAA** [comment]
@RussTreadon-NOAA No, I still need to submit these fixes I'm afraid. I'll make sure to do it by the end of the year.

**Comment by RussTreadon-NOAA** [comment]
Thank you @ClaraDraper-NOAA for your reply.  We'll keep this issue open until the changes are merged into `develop`.
